lib/archiving.js

```diff
@@ -191,13 +191,12 @@ function Archive(config, properties) {
 
 api = function (config, method, path, body, callback) {
   var rurl = config.apiEndpoint + '/v2/project/' + config.apiKey + path;
-  request.defaults(_.pick(config, 'proxy', 'timeout'))(
-    {
-      url: rurl,
-      method: method,
-      headers: generateHeaders(config),
-      json: body
-    },
+  config.headers = generateHeaders(config);
+  request.defaults(_.pick(config, 'proxy', 'timeout'));
```

Comment:
@manchuck The failing tests all have this message:

```
Error: Nock: No match for request {
      "method": "post",
      "url": "https://api.opentok.com/v2/project/APIKEY/archive",
      "headers": {
        "user-agent": "OpenTok-Node-SDK/2.16.0",
        "x-opentok-auth": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJBUElLRVkiLCJpc3QiOiJwcm9qZWN0IiwiaWF0IjoxNjk3NjcwOTM3LCJleHAiOjE2OTc2NzEyMzd9.2Ja75MtjsYP4cLvt-uYfXfUhIs07np0SLc9CbuZ2yUY",
        "accept": "application/json",
        "content-length": 190,
        "content-type": "application/x-www-form-urlencoded",
        "host": "api.opentok.com"
      },
      "body": "sessionId=1_MX4xMDB-MTI3LjAuMC4xflR1ZSBKYW4gMjggMTU6NDg6NDAgUFNUIDIwMTR-MC43NjAyOTYyfg&name=test_archive_name&hasAudio=&hasVideo=&outputMode=&layout=&resolution=&streamMode=&multiArchiveTag="
    }
```
Do we need to change something in these lines?